grails-app/services/org/broadinstitute/orsp/StatusEventService.groovy

```diff
@@ -56,69 +61,50 @@ class StatusEventService {
         // Terminal Event Case:
         // If the event is a terminal event (i.e. completed, abandoned, etc.) then the duration is
         // calculated from the first event to the terminal state event.
-        List<StatusEventDTO> eventDTOs = statusEvents.
-                withIndex().
+        List<StatusEventDTO> eventDTOs = statusEvents?.
+                withIndex()?.
```

Comment:
Not sure you need the `?`  in `withIndex` ... that's a built-in method on the groovy `Collection` class. Using elvis on `statusEvents` makes sense. But if it exists, then it has `withIndex` innately.